src/sql/parts/notebook/models/clientSession.ts

```diff
@@ -97,6 +99,11 @@ export class ClientSession implements IClientSession {
 			}
 			if (this._kernelPreference && this._kernelPreference.shouldStart) {
```

Comment:
Can we remove this (and remove the associated test 😄)? I'm like 99.9% sure we don't use kernelPreference any more.

Reply:
@kevcunnane, are you ok to remove it? I am not sure the initial usage for this. So i back and forth few times.

Reply:
If it's not used, get rid of it.

Reply:
great! Happy to removed it:)